src/omotes_simulator_core/entities/network_controller.py

```diff
@@ -138,6 +156,21 @@ def _set_producers_to_max(self) -> dict:
         producers[self.producers[0].id][PROPERTY_SET_PRESSURE] = True
         return producers
 
+    def _get_basic_producers(self) -> dict:
+        """Method to get the basic dict with setting for the producers.
+
+        :return dict: Dict with key= asset-id and value=setpoints for the producers.
+        """
+        producers = {}
+        for source in self.producers:
+            producers[source.id] = {
```

Comment:
This could also be a list of DataClasses with the respective variables: id, heat_demand, temp_r, temp_s, set_press_bool

Reply:
That woudl indeed be nicer to create a dataclass which is past back for the controller and then used by the assets. Issue #203